examples/detection/README.md

```diff
@@ -0,0 +1,109 @@
+
+# Before start
+
+0. Please install `mean-average-precision` python package (`pip install mean-average-precision`)
+1. Create or convert existing one training and validation datasets to [COCO format](https://cocodataset.org/#format-data) - you need to have `.json` files with annotations.
+
+In this example will be used dataset from [Kaggle - Fruit Detection](https://www.kaggle.com/andrewmvd/fruit-detection).
+
+Ton convert it to COCO you need to use following python script
+
+
+<details>
+<summary><bald>to_coco.py</bald></summary>
```

Comment:
what do you think about placing it as a `.py` file? so it could be much easier to use it if needed. and if it is not required... does it really matter if we have 1 extra `.py` file?